### PR TITLE
Fix PR4: two overlooked instances where material should have been specified

### PR DESCRIPTION
**Pull request by JoranAngevaare**
In previous PR https://github.com/JelleAalbers/wimprates/pull/4 (many thanks for the elaborate and instructive feedback), I overlooked two places when adding the 'material' option. This underestimated the Germanium and Argon rates as the [mediator](https://github.com/JelleAalbers/wimprates/pull/7/files#diff-9ef4e1501d49ecff72e0205410cd1dd5R168) and [pre-factor](https://github.com/JelleAalbers/wimprates/pull/7/files#diff-9ef4e1501d49ecff72e0205410cd1dd5R224) were referring to the Xenon-mass.

**Small check**
To test that I this time I did not mist no other instances (for SI heavy mediator coupling) I temporary set:
```
ATOMIC_WEIGHT = dict(
    Xe=None,
    Ar=39.948,
    Ge=72.64)
```
Thereafter, I loaded a spectrum for 'Ar' using ``wr.rate_wimp_std(..., **kwargs)``. With ``kwargs = {'material': 'Ar'}``. Does did not lead to a ``TypeError`` like it does for loading a 'Xe'-detection material:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
...
---> 23     return ATOMIC_WEIGHT[material] * nu.amu
...
TypeError: unsupported operand type(s) for *: 'NoneType' and 'float'
```

### CODE REVIEW

**Comment by JoranAngevaare** [comment]
@JelleAalbers this one is also ready to merge, although you might want to first merge https://github.com/JelleAalbers/wimprates/pull/9 in order to get a cleaner history.

This PR builds on https://github.com/JelleAalbers/wimprates/pull/9, where CI was added. You probably want to squash this PR and merge https://github.com/JelleAalbers/wimprates/pull/9, but that's up to you

**Comment by JelleAalbers** [comment]
Looks like there is some duplication between this one and #9, but the comparison does not show it. Maybe rebasing this onto master will fix that?

I see you bumped the version to 0.4.0 and 0.4.1 here. Is the intermediate version important for your projects? Otherwise it would probably be cleaner to merge all your changes, not change the version in this PR, then separately bump to 0.4.0 via a commit&tag to master.

**Comment by JoranAngevaare** [comment]
@JelleAalbers yeah I agree - I've rebased and made some tweaks (for completeness I put it in a separate PR: https://github.com/JoranAngevaare/wimprates/pull/9)

File: wimprates/migdal.py [hunk 1]
```diff
@@ -70,12 +111,15 @@ def rate_migdal(w, mw, sigma_nucleon, interaction='SI', m_med=float('inf'),
     include_approx_nr = 1 if include_approx_nr else 0
 
     result = 0
+    df_migdal, binding_es_for_migdal = read_migdal_transitions(material=material)
+    if consider_shells is None:
+        consider_shells = _default_shells(material)
     for state, binding_e in binding_es_for_migdal.items():
         binding_e *= nu.eV
         # Only consider n=3 and n=4
```
Comment (JelleAalbers) [inline wimprates/migdal.py]: You'll probably want to move this comment up inside _default_shells just before `Xe=['3*', '4*'],` (for other materials you consider different shells)

File: wimprates/migdal.py [hunk 2]
```diff
@@ -87,33 +131,38 @@ def rate_migdal(w, mw, sigma_nucleon, interaction='SI', m_med=float('inf'),
         def diff_rate(v, erec):
             # Observed energy = energy of emitted electron
             #                 + binding energy of state
-            eelec = w - binding_e - include_approx_nr * erec * 0.15
+            eelec = w - binding_e - include_approx_nr * erec * q_nr
             if eelec < 0:
+                # print(w , binding_e , include_approx_nr * erec * 0.15)
```
Comment (JelleAalbers) [inline wimprates/migdal.py]: Some leftover commented code here

**Review by JelleAalbers — APPROVED** [review]
Thanks Joran, looks great as usual! Added two minor comments, but I can also fix those myself after merging if you prefer. After the merge let's release 0.4.0.

**Comment by JoranAngevaare** [comment]
Thanks Jelle! I removed some of the scaffolding:
https://github.com/JelleAalbers/wimprates/pull/7/commits/e436268a345c7e28068e009d642ab433a8261c72

**Review by JelleAalbers — APPROVED** [review]

**Comment by JelleAalbers** [comment]
Thanks Joran, I'll squash merge as you suggested (e.g. to avoid the version going back and forth) then release 0.4.0

**Pull request merged**